### DISCUSSION TITLE
Prevent neo-tree from opening files in edgy windows

### DISCUSSION THREAD

**Discussion by folke** [folke/edgy.nvim#4] [Show and tell]
```lua
  -- prevent neo-tree from opening files in edgy windows
  {
    "nvim-neo-tree/neo-tree.nvim",
    optional = true,
    opts = {
      open_files_do_not_replace_types = { "terminal", "Trouble", "qf", "edgy" },
    },
  },
```

**Comment by patricorgi** [comment]
With this snippet, I'm able to prevent neo-tree from replacing certain buftypes. Thanks!

I'm experiencing similar issue with telescope. When my cursor is placed in a buffer with the type `OverseerList` and run telescope find_files command, the opened file will replace the OverseerList buffer. Is there a way to use edgy to force telescope to open file in the main editor?

**Comment by weakphish** [comment]
Is it possible to make this part of the default configuration for LazyVim when neo-tree is enabled?
